stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -248,6 +248,18 @@ end
     end
 end
 
+@testset "normalize for multidimensional arrays" begin
+    arr = [ [1.0 0.0]; [0.0 1.0] ]
```

Comment:
This looks a bit like numpy but it is not how you do it in julia. You can simply create a `Matrix` like e.g. `[a b; c d]`.

Reply:
updated in f602363